packages/react/src/patterns/sub-patterns/TableOfContents/__stories__/TableOfContents.stories.js

```diff
@@ -1,23 +1,33 @@
-/* eslint-disable jsx-a11y/anchor-is-valid */
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
 import './index.scss';
-import { select, text, withKnobs } from '@storybook/addon-knobs';
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import dataContent from './data/dataContent';
 import { DDS_TOC } from '../../../../internal/FeatureFlags';
 import React from 'react';
 import readme from '../README.md';
 import { storiesOf } from '@storybook/react';
 import TableOfContents from '../TableOfContents';
 
 if (DDS_TOC) {
```

Comment:
@carthick almost missed this. Should remove the feature flag here.

Reply:
Also, need to remove `DDS_TOC` overall (in `FeatureFlags.js`, etc)

Reply:
@jeffchew removed and updated Feature Flag... Please let me know if anything needs to be changed